packages/data-service-generator/src/plugin-wrapper.ts

```diff
@@ -83,16 +83,21 @@ const pluginWrapper: PluginWrapper = async (
     context.modules.push(finalModules);
     return finalModules;
   } catch (error) {
-    context.logger.error(`failed to execute plugin event ${event}`, {
-      errorMessage: JSON.stringify((error as Error).message),
+    const friendlyErrorMessage = `Failed to execute plugin event ${event}`;
+
+    context.logger.error(friendlyErrorMessage, {
+      error: { message: error.message, stack: error.stack },
```

Comment:
do we want to write the entire stack to the build log?

Reply:
`context.logger.error` is for application logs, `await createLog(` is for build logs visible on the client